frontend/components/site/siteBuildLogs.jsx

```diff
@@ -38,10 +38,15 @@ function getSiteBuildLogTable(buildDetails, logs, state) {
   );
 }
 
+function hasBuildTasks(buildTasks) {
+  return process.env.FEATURE_BUILD_TASKS === 'active' && buildTasks?.length > 0;
```

Comment:
Does this account for a site branch not having any build tasks associated to it?